README.rst

```diff
@@ -98,6 +98,7 @@ Besides the numerical argument, there are two main optional arguments.
 * ``ja`` (Japanese)
 * ``kn`` (Kannada)
 * ``ko`` (Korean)
+* ``kr`` (Kinyarwanda)
```

Comment:
The official [ISO 639-1 code for Kinyarwanda is "rw"](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) so we should use that, since "kr" is reserved for Kanuri (a Sudanese language).

Reply:
Fixed!!

Reply:
thanks!